gen/dcos-config.yaml

```diff
@@ -384,8 +385,13 @@ package:
               "network_mode": "USER",
               "dns": {
                 "nameservers": [
+{% switch enable_ipv6 %}
+{% case "true" %}
                                 "198.51.100.1",
                                 "fd01:d::c633:6401"
```

Comment:
Just a minor comment:
Can't we do 
"198.51.100.1",
{%switch enable_ipv6%}
{%case false%}
{%case true%}
"fd01:d::c633:6401"
{%endswitch%}

Reply:
See the comma. I tried to do like below, but it looks ugly for me.

```
"198.51.100.1"
{%switch enable_ipv6%}
{%case false%}
{%case true%}
, "fd01:d::c633:6401"
{%endswitch%}
```

Reply:
👍